server/aws-lsp-codewhisperer/src/language-server/qChatServer.ts

```diff
@@ -1,39 +1,110 @@
+import {
+    GenerateAssistantResponseCommandInput,
+    GenerateAssistantResponseCommandOutput,
+} from '@amzn/codewhisperer-streaming'
+import { ChatParams, ErrorCodes, ResponseError } from '@aws/language-server-runtimes/protocol'
```

Comment:
small note: as a server developer, you should be able to rely on types solely from `@aws/language-server-runtimes/server-interface`. `/protocol` subpackage purpose to define protocol for client communication, which is that proxied to server through `server-interface` shapes transiently.

Reply:
Great to know, I will keep that in mind. I have updated the import location.